itl/command-plugins.conf

```diff
@@ -340,6 +340,10 @@ object CheckCommand "ssl" {
 				}
 			}
 		}}
+    "-N" = {
+      value = "$ssl_sni$"
+      description = "Enable SSL/TLS hostname extension support (SNI)"
+    }
```

Comment:
Please fix the indent (tabs) here :)

Reply:
Didn't notice the wrong indent her, in my vim editor all looked good. I found out that this was a caused my some of my .vimrc settings. Anyway the indent should be fine now :)